coachmarks/src/main/java/com/myntra/coachmarks/ui/presenter/PopUpCoachMarkPresenter.java

```diff
@@ -60,6 +61,8 @@
     private IStringResourceProvider mStringResourceProvider;
     private IDimensionResourceProvider mDimensionResourceProvider;
 
+    private boolean mDefaultPopUpLocationFailed = false;
+
```

Comment:
Well, I would try to refrain from having these kind of state. Can try to be stateless ?